example-ideal.txt

```diff
@@ -0,0 +1,17 @@
+from findBottomRightCorner in Empty Empty Empty South to West
+from findBottomRightCorner in Empty Empty West South to North
+from findBottomRightCorner in North Empty West Empty to East
+from findBottomRightCorner in Empty Empty West Empty to North
+from findBottomRightCorner in Empty Empty Empty Empty to North
+from findBottomRightCorner in North Empty Empty Empty to East
+from findBottomRightCorner in North East Empty Empty to South
+from findBottomRightCorner in Empty East Empty Empty to South
+from findBottomRightCorner in Empty East Empty South to West in goNorth
+from goNorth in Empty Empty Empty South to North
+from goNorth in Empty Empty Empty Empty to North
+from goNorth in North Empty Empty Empty to West in goSouth
+from goSouth in North Empty Empty Empty to South
+from goSouth in Empty Empty Empty Empty to South
+from goSouth in Empty Empty Empty South to West in goNorth
+from goSouth in North Empty West Empty to South
+from goSouth in Empty Empty West Empty to South
```

Comment:
If this was the original syntax you were aiming to implement, I can see how it changed. I think that I honestly might like what you ended up with better. However, I do like the ability for a user to specify their destination like findBottomRightCorner